### DISCUSSION TITLE
Mountainduck some files & folders disappear in Finder temporarily

### DISCUSSION THREAD

**Discussion by svenniemann** [iterate-ch/cyberduck#16106] [Q&A] [answered]
Hi,
I'm using Mountainduck to mount my Debian NAS via SFTP in Mac OS (14.5).

It mostly works fine, but sometimes randomly files/folders disappear for some time. They are still there on the NAS, they are just not visible in Finder or Terminal on the Mac. After some time they appear again. 

Any ideas why this is happening. It is really annoying.

**Comment by tatze96** [comment]
This sounds like your synchronization gets paused temporarily and was resumed at the point where the files re-appear.

Please enable [debug logging](https://docs.cyberduck.io/mountainduck/support/#debug-log), recreate the issue and send us the [log files](https://docs.cyberduck.io/mountainduck/support/#logging-output) to support@mountainduck.io with the ticket reference GH-16106 for further investigation.

Some general information regarding paused synchronization:
The synchronization can be paused manually but also gets paused by Mountain Duck if the connection to the server is lost or too unstable to transfer files. Files and folders that weren't indexed before the server or network connection dropped won't show up if you work offline.

In the case of Mountain Duck pausing the synchronization process due to connectivity issues, Mountain Duck performs reachability checks at regular intervals and will resume the synchronization automatically once the connection can be established again.

**Reply by svenniemann** [reply]
thanks for your feedback!

The paused synchronization issue you describe shouldn't be relevant if using online mode I guess? I can understand this might happen in smart sync mode, but should not be an issue in online mode. If in online mode there is a connection issue I would expect nothing to be visible.

Didn't happen again yet.  Once it happened again I will send the logs.

**Reply by tatze96** [reply]
Correct. If the connection is interrupted within the *Online* Mode, Mountain Duck will display a "Disconnect" error message and subsequently disconnect the drive.

**Reply by dkocher** [reply]
Documented in [Connectivity Issues](https://docs.mountainduck.io/mountainduck/connect/online/#connectivity-issues).

**Reply by svenniemann** [reply]
@dkocher no, this is not documented. It is not unmounted. Only some folders/files are disappering. Other folders/files from the same share are completely fine and accessible

**Comment by svenniemann** [comment, marked as answer]
After sharing logs via mail it was suggested to switch to current Snapshot build, since the issue should be already fixed there.
Issue didn't happen yet since I updated. So it looks like this is indeed solved :)
